samples/nrf_rpc/ps_client/src/log_rpc_shell.c

```diff
@@ -11,7 +11,29 @@
 
 #include <stdlib.h>
 
-static int log_rpc_get_crash_log_cmd(const struct shell *sh, size_t argc, char *argv[])
+static int cmd_log_rpc_stream_level(const struct shell *sh, size_t argc, char *argv[])
+{
+	int rc = 0;
+	enum log_rpc_level level;
+
+	level = (enum log_rpc_level)shell_strtol(argv[1], 10, &rc);
+
+	if (rc) {
+		shell_error(sh, "Invalid argument: %d", rc);
+		return rc;
+	}
+
+	rc = log_rpc_set_stream_level(level);
+
+	if (rc) {
+		shell_error(sh, "Error: %d", rc);
+		return rc;
```

Comment:
I'm not sure that it is correct to return any value in case of error. Public API says:
```
/**
 * @brief Shell command handler prototype.
 *
 * @param sh Shell instance.
 * @param argc  Arguments count.
 * @param argv  Arguments.
 *
 * @retval 0 Successful command execution.
 * @retval 1 Help printed and command not executed.
 * @retval -EINVAL Argument validation failed.
 * @retval -ENOEXEC Command not executed.
 */
```
https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/shell/shell.h#L162-L175
I guess it should be `-ENOEXEC`.
Here and below.

Reply:
Done.

Reply:
thanks, but, I guess, `cmd_log_rpc_crash` requires the same change, doesn't?

Reply:
right, sorry, done.